source/section_3_13.rst

```diff
@@ -59,7 +59,10 @@ of increased risk despite contract security provisions.
 | System communication boundaries are defined across fixed-function networking     |
 | devices like routers and firewalls, and VPNs and related tunnel technologies.    |
 | Customers retain all responsibility for configuration and maintenance of these   |
-| devices.                                                                         |
+| devices. NILRT+SNAC devices are equipped with a software firewall, firewalld,    |
+| that provides protection to the device and prevents the device from emitting     |
+| packets with a spoofed source address. Additional configuration of firewalld can |
```

Comment:
Does our firewalld configuration protect against source address spoofing by default? Or does the admin have to configure something?